development/extract_strings_from_objects.py

```diff
@@ -186,8 +185,8 @@ def add_line(value, component, sep, key, filename):
 	strings = sorted(object_strings)
 
 	if strings:
-		return(u'\n\t"{0!s}" : {{{1!s}{2!s},{3!s}}'.
```

Comment:
So you remove the unicode marker here, but add it in the above diff and remove it in the one above that one. Can you explain, why?